experiment_buddy/experiment_buddy.py

```diff
@@ -249,40 +251,37 @@ def _open_ssh_session(hostname: str) -> fabric.Connection:
     return ssh_session
 
 
-def _ensure_scripts(hostname: str, extra_slurm_header: str, working_dir: str) -> Tuple[str, fabric.Connection]:
-    ssh_session = _open_ssh_session(hostname)
+def ensure_scripts(ssh_session: fabric.Connection, extra_slurm_header: str, working_dir: str) -> str:
     retr = ssh_session.run("mktemp -d -t experiment_buddy-XXXXXXXXXX")
-    remote_tmp_folder = retr.stdout.strip() + "/"
+    remote_tmp_folder = f'{retr.stdout.strip()}/'
```

Comment:
would have been a good change, but don't change what is not necessary.
for all these kind of style changes do a single PR doing only visual changes.

Reply:
True, but I'd prefer to have either all string concatenation or string template across the whole project, the change is minimal and managed automatically by the IDE

Would it be better to have a separated PR only for this?

Reply:
Yes please, a pr dedicated to style changes in general is also good

Reply:
Moved